Config.psm1

```diff
@@ -21,57 +21,87 @@ Import-Module "$localResourcesDir\ini.psm1"
         @{"Name" = "drivers_path"; "GroupName" = "drivers";
-          "Description" = ""},
+          "Description" = "The location where additional drivers that are needed for the image are located."},
         @{"Name" = "install_updates"; "GroupName" = "updates"; "DefaultValue" = $false; "AsBoolean" = $true;
-          "Description" = ""},
+          "Description" = "If set to $true, the latest updates will be downloaded and installed."},
         @{"Name" = "purge_updates"; "GroupName" = "updates"; "DefaultValue" = $false; "AsBoolean" = $true;
-          "Description" = ""},
+          "Description" = "If set to true, will run DISM with /resetbase option. This will reduce the size of
+                           WinSXS folder, but after that Windows updates cannot be uninstalled."},
         @{"Name" = "run_sysprep"; "GroupName" = "sysprep"; "DefaultValue" = $true; "AsBoolean" = $true;
-          "Description" = ""},
+          "Description" = "Used to clean the OS on the VM, and to prepare it for a first-time use."},
         @{"Name" = "unattend_xml_path"; "GroupName" = "sysprep"; "DefaultValue" = "UnattendTemplate.xml";
-          "Description" = ""},
+          "Description" = "The path to the Unattend XML template file used for sysprep."},
         @{"Name" = "disable_swap"; "GroupName" = "sysprep"; "DefaultValue" = $false; "AsBoolean" = $true;
-          "Description" = ""},
+          "Description" = "DisableSwap option will disable the swap when the image is generated and will add a setting
+                           in the Unattend.xml file which will enable swap at boot time during specialize step.
+                           This is required, as by default, the amount of swap space on Windows machine is directly
+                           proportional to the RAM size and if the image has in the initial stage low disk space,
+                           the first boot will fail due to not enough disk space. The swap is set to the default
+                           automatic setting right after the resize of the partitions is performed by cloudbase-init."},
         @{"Name" = "persist_drivers_install"; "GroupName" = "sysprep"; "DefaultValue" = $true; "AsBoolean" = $true;
-          "Description" = ""},
+          "Description" = "In case the hardware on which the image is generated will also be the hardware on
+                           which the image will be deployed this can be set to true, otherwise the spawned
+                           instance is prone to BSOD."},
         @{"Name" = "beta_release"; "GroupName" = "cloudbase_init"; "DefaultValue" = $false; "AsBoolean" = $true;
-          "Description" = ""}
+          "Description" = "This is a switch that allows the selection of Cloudbase-Init branches. If set to true, the
+                           beta branch will be used:
+                           https://cloudbase.it/downloads/CloudbaseInitSetup_x64.msi
```

Comment:
use https://cloudbase.it/downloads/CloudbaseInitSetup_<arch>.msi, where arch can be x86 or x64